test/notification_test.go

```diff
@@ -12,17 +13,23 @@ func TestAssertNotificationSent(t *testing.T) {
 	n := fyne.NewNotification("Test Title", "Some content")
 	myApp := fyne.CurrentApp()
 
-	AssertNotificationSent(t, n, func() {
+	test.AssertNotificationSent(t, n, func() {
 		fyne.CurrentApp().SendNotification(n)
 	})
 	assert.Equal(t, myApp, fyne.CurrentApp())
 }
 
+func TestAssertNotificationSent_Nil(t *testing.T) {
+	test.AssertNotificationSent(t, nil, func() {
+		// don't send anything
+	})
+}
+
 func TestAssertNotificationSent_NotSent(t *testing.T) {
-	myApp := fyne.CurrentApp()
+	tt := &testing.T{}
 
-	AssertNotificationSent(t, nil, func() {
+	test.AssertNotificationSent(tt, &fyne.Notification{}, func() {
 		// don't send anything
 	})
-	assert.Equal(t, myApp, fyne.CurrentApp())
+	assert.True(t, tt.Failed(), "A notification was sent incorrectly")
```

Comment:
The explanation confuses me. We test the notification assert helper. If this `assert.True` fails, we assume that the helper is broken. This means that no notification was sent (as expected) but that the helper did something wrong.
Maybe the explanation should be like “asserting notification should fail if no notification was sent”.